src/sentry/runner/commands/run.py

```diff
@@ -528,11 +530,30 @@ def ingest_consumer(consumer_types, all_consumer_types, **options):
 
 @run.command("ingest-metrics-consumer")
 @log_options()
-@click.option("--topic", default="ingest-metrics", help="Topic to get subscription updates from.")
+@click.option("--topic", default="ingest-metrics", help="Topic to get metrics data from.")
 @batching_kafka_options("ingest-metrics-consumer")
 @configuration
 def metrics_consumer(**options):
 
     from sentry.sentry_metrics.indexer.indexer_consumer import get_metrics_consumer
 
     get_metrics_consumer(**options).run()
+
+
+@run.command("ingest-metrics-consumer-2")
+@log_options()
+@click.option("--topic", default="ingest-metrics", help="Topic to get metrics data from.")
+@batching_kafka_options("ingest-metrics-consumer")
+@configuration
+@click.option(
+    "--processes",
+    default=1,
+    type=int,
+)
+@click.option("--input-block-size", type=int, default=DEFAULT_BLOCK_SIZE)
+@click.option("--output-block-size", type=int, default=DEFAULT_BLOCK_SIZE)
+def metrics_streaming_consumer(**options):
+    from sentry.sentry_metrics.multiprocess import get_streaming_metrics_consumer
+
+    streamer = get_streaming_metrics_consumer(**options)
```

Comment:
What is the batch size we set in prod? I think we should start with 100.
There will be a competitive dynamic here we need to tune:
- bigger batch: we get the most out of batching
- too big batch: probably the batches are coutnerproductive
- smaller batches: more chances to parallelize them. 
- too small batches: the processing takes too short and we do not use parallelism enough 
- too big batches: we spend all our time building batches for the multi processing and we end up using only one process.

I think 100 may be a good start. Let's ensure that it is clear to everybody involved when you deploy for the first time that we will need tuning.